src/components/content/CardPhone.jsx

```diff
@@ -1,29 +1,34 @@
 /* eslint-disable react/prop-types */
 
-import { Link } from "react-router-dom"
+import { Link } from "react-router-dom";
 
-
-export const CardPhone = ({idPhone, name, brand, ram, rom, price, img }) => {
+export const CardPhone = ({ idPhone, name, brand, ram, rom, price, img }) => {
```

Comment:
why do you have this file? you created another one with the same name but with TypeScript extension

Reply:
it was my error because i just have change the extension but for error i created another archive and i added the typescript but after i only added the extensions to others archives